claim/test_services.py

```diff
@@ -193,55 +193,6 @@ def test_extended_claim_submit_xml(self):
         expected = expected + "</Claim>"
         self.assertEquals(expected, claim.to_xml())
 
-    @mock.patch('django.db.connections')
```

Comment:
Should we be removing tests completely? At maximum they could be skipped or commented out for now in my opinion. If they require bigger rewrite we should postpone this, not remove them.
@dragos-dobre @dborowiecki